Example-iOS-Swift/Events.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Events.swift
+//  Example-iOS-Swift
+//
+//  Created by Max Andreassen on 28/09/2023.
+//  Copyright © 2023 Wolf3D. All rights reserved.
```

Comment:
minor but we should replace all Wolf3D stuff with Ready Player Me

Reply:
Good idea, will do.